index.html

```diff
@@ -74,12 +74,12 @@
 				<div class="card-action">
 					<div class="audiosetting">
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="previousBackgroundAudio();" title="Previous Audio"><i class="fa fa-step-backward" aria-hidden="true"></i></a>
-						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="nextBackgroundAudio();" title="Next Audio"><i class="fa fa-step-forward" aria-hidden="true"></i></a>
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="startBackgroundAudio();" title="Play Audio"><i class="fa fa-play" aria-hidden="true"></i></a>
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="stopBackgroundAudio();" title="Stop"><i class="fa fa-pause" aria-hidden="true"></i></a>
+						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="nextBackgroundAudio();" title="Next Audio"><i class="fa fa-step-forward" aria-hidden="true"></i></a>
 					</div>
 					<p class="range-field">
-						<input type="range" onchange="setVolume()" id='volume1' min=0 max=1 step=0.01 value='1'>
+						<input type="range" onchange="setVolume()" id='volume1' min=0 max=100 step=1 value='100'>
```

Comment:
Why did you change that? It was for volume. And volume works on from range of 0-1 in js. There was a reason it was put there. Stop doing useless changes.

Reply:
ohkk :D reverting that then :D